inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -166,13 +166,14 @@ def get_commands(self):
                                               COMMANDS_TEMPLATE_FILE_NAME)
 
         with tempfile.NamedTemporaryFile() as commands_file:
+            logging.info(commands_file.name)
             replace_params_in_template(
                 template_path=commands_template_path,
                 params={"n_cores": self.n_cores},
                 output_file_path=commands_file.name,
             )
 
-            commands = self.read_commands_from_file(commands_file)
+            commands = self.read_commands_from_file(commands_file.name)
```

Comment:
This does not seem to be working. 

`TypeError: expected str, bytes or os.PathLike object, not _TemporaryFileWrapper`

Reply:
I confirmed it works on my side. This is likely because your local version isn't updated.